pandas/io/formats/style.py

```diff
@@ -1441,7 +1441,7 @@ def relative_luminance(rgba) -> float:
                     The relative luminance as a value from 0 to 1
                 """
                 r, g, b = (
-                    x / 12.92 if x <= 0.03928 else ((x + 0.055) / 1.055 ** 2.4)
+                    x / 12.92 if x <= 0.04045 else ((x + 0.055) / 1.055) ** 2.4
```

Comment:
can you point to a reference where this calculation is

Reply:
the doc string for `relative_luminance` already does in fact - how I found out it was wrong and out-of-date. Here fyi:  https://www.w3.org/WAI/GL/wiki/Relative_luminance.